src/Pages/ReservationList/ReservationList.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from "react";
+import Reservation from "./Reservation";
+import "./ReservationList.scss";
+
+class ReservationList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      reservation: [],
+    };
+  }
+
+  componentDidMount = () => {
+    const token = localStorage.getItem("token");
+    if (token) {
+      fetch("http://192.168.219.108:8001/reservation/get", {
```

Comment:
만약에 호현님, 용민님 api로 나뉜다면 
```API_HH```, ```API_YM```
이런식으로 변수 설정해서 config 파일 안에서만 서버 주소 변경하면 다 바뀔 수 있게 해주세요!
```suggestion
      fetch(`${API}/reservation/get`, {
```

Reply:
네 수정했습니다.